webapp/src/ddocs/medic-tombstone/views/tombstones_by_doc_id/map.js

```diff
@@ -0,0 +1,8 @@
+function(doc) {
+  if (doc.type !== 'tombstone') {
+    return;
+  }
+
+  var matches = doc._id.match(/^(.*)____(.*)____tombstone$/);
+  emit(matches[1], { rev: matches[2] });
+}
```

Comment:
Is this used anywhere? I can't find a reference. It'd be better to remove it and generate the tombstone ID in code then request that directly rather than having another view to maintain.